src/client/components/forms/publisher.js

```diff
@@ -55,21 +72,32 @@ class PublisherForm extends React.Component {
 		});
 	}
 
+	/**
+	 * Handler to direct the user to the previous tab.
+	 */
 	handleBackClick() {
 		this.handleTabSelect(this.state.tab - 1);
 	}
 
+	/**
+	 * Handler to direct the user to the next tab.
+	 */
 	handleNextClick() {
 		this.handleTabSelect(this.state.tab + 1);
 	}
 
+	/**
+	 * Handler to send publisher information in the form to the server.
+	 * @param {Event} evt - used to prevent event propagation
```

Comment:
`@param {Event} evt - The submission event that triggered the handler` would be better, I think.